controller/jobController.ts

```diff
@@ -23,14 +23,16 @@ module.exports = function(app: Application){
 
     app.get ("/list-jobs/:id", async (req: Request, res: Response) => {
         let data: Job;
+        let data2: Responsibility[];
```

Comment:
Could we maybe call this something better than data2?

Reply:
@eoinmcbennett updated to be called 'let dataRole: Responsibility[];'